src/utils/error.ts

```diff
@@ -51,7 +51,8 @@ export default class TeaError extends Error {
 
         open a pull-request to add it to the pantry:
 
-            https://github.com/teaxyz/pantry.extra
+            https://github.com/teaxyz/pantry.zero#contributing
```

Comment:
Might want to modify line 52 to include `. see` before the colon, so we don't get new package PRs to pantry.zero.